tests/requirements.test.txt

```diff
@@ -17,3 +17,4 @@ torch
 pytest
 coverage
 paramiko==2.7.2
+pytz==2019.3
```

Comment:
when we use it?

Reply:
force replace the newer version, the tests only work when pytz < 2020